homeassistant/components/fullykiosk/manifest.json

```
@@ -0,0 +1,13 @@
+{
+  "domain": "fullykiosk",
```

Shouldn't the domain be:

```suggestion
  "domain": "fully_kiosk",
```

?